klog.go

```diff
@@ -1223,26 +1223,39 @@ func StartFlushDaemon(interval time.Duration) {
 // lockAndFlushAll is like flushAll but locks l.mu first.
 func (l *loggingT) lockAndFlushAll() {
 	l.mu.Lock()
+	// Write the klog buffer into the kernel buffer
 	l.flushAll()
 	l.mu.Unlock()
+	// Write the kernel buffer to disk
+	l.syncAll()
 }
 
-// flushAll flushes all the logs and attempts to "sync" their data to disk.
+// FlushAll flushes all the logs
```

Comment:
`flushAll` NOT `FlushAll`

Reply:
Thank you for taking the time to help review the code. I have finished making the modifications.